save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/SelectForm.kt

```diff
@@ -0,0 +1,129 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS", "HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package org.cqfn.save.frontend.components.basic
+
+import org.cqfn.save.entities.Organization
+import org.cqfn.save.frontend.utils.*
+
+import org.w3c.dom.events.Event
+import org.w3c.fetch.Headers
+import react.Props
+import react.dom.*
+import react.fc
+import react.useState
+
+import kotlinx.html.js.onChangeFunction
+
+/**
+ * SelectFormRequired component props
+ */
+external interface SelectFormRequiredProps : Props {
+    /**
+     * Type of 'select'
+     */
+    var form: InputTypes?
+
+    /**
+     * Flag to valid select
+     */
+    var validInput: Boolean?
+
+    /**
+     * classes of 'select'
+     */
+    var classes: String?
+
+    /**
+     * select name
+     */
+    var text: String?
+
+    /**
+     * elements of `select`
+     */
+    var elements: List<String>?
+
+    /**
+     * initial value of `select`
+     */
+    var initialValue: String?
```

Comment:
Are these still used? You don't set them from parent, so they're always null?

Reply:
Yes. Removed it

Reply:
And what about `elements`?

Reply:
Fixed